costmap_cspace/include/costmap_cspace/costmap_3d_layer/base.h

```diff
@@ -365,7 +365,15 @@ class Costmap3dLayerBase
     map_updated_ = msg;
 
     region_ = UpdatedRegion(ox, oy, 0, w, h, map_->info.angle, msg->header.stamp);
-    updateChainEntry(UpdatedRegion(ox, oy, 0, w, h, map_->info.angle, msg->header.stamp));
+
+    if (update_chain_entry)
+    {
+      updateChainEntry(UpdatedRegion(ox, oy, 0, w, h, map_->info.angle, msg->header.stamp));
+    }
+    else
+    {
+      ROS_INFO("update_chain_entry execution has been avoided.");
```

Comment:
I think it should be debug level or no-log

Reply:
debug ni kaemasu.